face_alignment/api.py

```diff
@@ -127,23 +132,28 @@ def get_landmarks_from_image(self, image_or_path, detected_faces=None, return_bb
 
         landmarks = []
         landmarks_scores = []
-        for i, d in enumerate(detected_faces):
+        for i, d in enumerate(detected_faces): 
             center = torch.tensor(
                 [d[2] - (d[2] - d[0]) / 2.0, d[3] - (d[3] - d[1]) / 2.0])
-            center[1] = center[1] - (d[3] - d[1]) * 0.12
-            scale = (d[2] - d[0] + d[3] - d[1]) / self.face_detector.reference_scale
+            if self.face_detector:
+                center[1] = center[1] - (d[3] - d[1]) * 0.12
+                scale = (d[2] - d[0] + d[3] - d[1]) / self.face_detector.reference_scale
+            else:
+                scale = (d[2]-d[0])/200
 
             inp = crop(image, center, scale)
             inp = torch.from_numpy(inp.transpose(
                 (2, 0, 1))).float()
-
+            
             inp = inp.to(self.device)
             inp.div_(255.0).unsqueeze_(0)
 
+            torch._C._set_graph_executor_optimize(False)

```

Comment:
we should explain why we are using this. And maybe link some page that explains why this work